pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/export/tabular/FilterExportConfig.java

```diff
@@ -53,7 +54,17 @@ public FilterExportConfig(ModuleExportConfig parent, Filter filterColumn, boolea
         List<Map<String, Object>> childQuestions = getChildQuestions();
         if (childQuestions != null) {
             this.childConfigs = IntStream.range(0, childQuestions.size())
-                    .mapToObj(qNum -> new FilterExportConfig(this, childQuestions.get(qNum), qNum)).collect(Collectors.toList());
+                    .mapToObj(qNum -> {
+                            FilterExportConfig filterExportConfig = new FilterExportConfig(this, childQuestions.get(qNum), qNum);
+                            if (filterExportConfig.getOptions() != null) {
+                                if (this.options == null) {
+                                    this.options = new ArrayList<>();
+                                }
+                                this.options.addAll(filterExportConfig.getOptions());
```

Comment:
adding the options from the childQuestion into options 
https://broadworkbench.atlassian.net/browse/PEPPER-551